.github/workflows/PROJECT-AUTO-CHANGELOG-CONTROL.yaml

```
@@ -287,17 +287,55 @@ jobs:
+          
+          # Make sure we have the latest from the base branch
+          git fetch origin $PR_BASE
+          
+          # Update PR head branch with latest from base branch
+          git merge --no-edit origin/$PR_BASE || true
           git push origin HEAD
 
       - name: 자동 PR Merge
         env:
           GH_TOKEN: ${{ secrets._GITHUB_PAT_TOKEN }}
         run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:22:25: Double quote to prevent globbing and word splitting

(shellcheck)